From 31a9a6ffc10f9737e70d7f0051ff590ff284ad07 Mon Sep 17 00:00:00 2001
From: Rico Tzschichholz <ricotz@ubuntu.com>
Date: Tue, 5 Sep 2017 00:01:16 +0200
Subject: Accept GnuPG 2.2.x as supported version

https://git.gnome.org/browse/seahorse/commit/?id=31a9a6ffc10f9737e70d7f0051ff590ff284ad07
https://git.gnome.org/browse/seahorse/patch/?id=31a9a6ffc10f9737e70d7f0051ff590ff284ad07

Note: had to change GPGME_REQUIRED=1.7.0 to GPGME_REQUIRED=1.0.0 for
3.20.0 for patch to apply

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index fc4493c..723b46f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9,7 +9,7 @@ GCR_REQUIRED=3.11.91
 GTK_REQ=3.4.0
 GTK_MAX=GTK_VERSION_3_4
 
-GNUPG_ACCEPTED="2.0.12 2.1.4"
+GNUPG_ACCEPTED="2.0.12 2.1.4 2.2.0"
 GPGME_REQUIRED=1.0.0
 LIBSECRET_REQUIRED=0.16
 AVAHI_GLIB_REQUIRED=0.6
-- 
cgit v0.12


